internal/services/postgres/postgresql_flexible_server_resource.go

```diff
@@ -382,7 +382,6 @@ func resourcePostgresqlFlexibleServer() *pluginsdk.Resource {
 			}
```

Comment:
Suggest run all related test cases to ensure this fix wouldn't introduce the regression issue.

Reply:
There is a regression issue on the existing test case.

------- Stdout: -------
=== RUN   TestAccPostgresqlFlexibleServer_geoRestore
=== PAUSE TestAccPostgresqlFlexibleServer_geoRestore
=== CONT  TestAccPostgresqlFlexibleServer_geoRestore
    testcase.go:113: Step 3/4 error: After applying this test step, the plan was not empty.
        stdout:
        Terraform used the selected providers to generate the following execution
        plan. Resource actions are indicated with the following symbols:
          ~ update in-place
        Terraform will perform the following actions:
          # azurerm_postgresql_flexible_server.geo_restore will be updated in-place
          ~ resource "azurerm_postgresql_flexible_server" "geo_restore" {
                id                                = "/subscriptions/*******/resourceGroups/acctestRG-postgresql-240514110942768666/providers/Microsoft.DBforPostgreSQL/flexibleServers/acctest-fs-restore-240514110942768666"
                name                              = "acctest-fs-restore-240514110942768666"
              + storage_tier                      = "P4"
                # (14 unchanged attributes hidden)
                # (1 unchanged block hidden)
            }
        Plan: 0 to add, 1 to change, 0 to destroy.
--- FAIL: TestAccPostgresqlFlexibleServer_geoRestore (2753.46s)
FAIL